hibernate-reactive-core/src/test/java/org/hibernate/reactive/configuration/JdbcUrlParserTest.java

```diff
@@ -125,7 +155,7 @@ private void assertOptions(String url, String expectedDbName, Map<String, String
 		assertThat( options.getUser() ).as( "URL: " + url ).isEqualTo( username );
 		assertThat( options.getPassword() ).as( "URL: " + url ).isEqualTo( password );
 		assertThat( options.getDatabase() ).as( "URL: " + url ).isEqualTo( expectedDbName );
-		assertThat( options.getHost() ).as( "URL: " + url ).isEqualTo( "localhost" );
+		assertThat( options.getHost() ).as( "URL: " + url ).isEqualTo( expectedHost );
 		assertThat( options.getPort() ).as( "URL: " + url ).isEqualTo( dbType().getDefaultPort() );
```

Comment:
We need to add some tests for checking that we read the correct port if set.

Reply:
Done